src/cppcache/src/TcrMessage.hpp

```diff
@@ -1170,7 +1170,7 @@ class TcrMessageHelper {
     }
     if (compId != expectedPartType) {
       char exMsg[256];
-      ACE_OS::snprintf(exMsg, 255,
+      std::snprintf(exMsg, 255,
```

Comment:
We don't have a good command to execute. I suggest looking for integration into what ever editor you have with clang-format. There appears to be some differences between clang-format on mac and windows so limit the formatting to files you are editing. 

For example, in Eclipse, I have reformat on save configured. 

The following lines used to be aligned with the changed line's open paren. The guide defines specific rules for when to align to the paren or indent.